mava/utils/jumanji_register.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+from typing import Any, Dict
+
+# File path to store the generated jumanji's scenarios
+_SCENARIOS_FILE_PATH = "../Mava/mava/configs/env/scenario/jumanji_scenarios.json"
```

Comment:
Yes that's better! I'll update the code and generate the config file with limited scenarios (the ones we use the most) then we stick with the check function.

Should we keep the storing in case a new scenario is given or just check and extract the new attributes (I'm afraid this will update the yaml file each time)?